Tests/GauntletTests/Operators/EqualWithAccuracyOperatorsTests.swift

```diff
@@ -56,7 +56,8 @@ class EqualWithAccuracyAssertsTestCase: XCTestCase {
         // Then
         Assert(that: assertion)
             .didFail(expectedName: "isEqualTo(, accuracy)", expectedLine: expectedLine)
-            .isEqualTo(.message("4 is not equal to the expected value 1. Accuracy: 2"))
+            .isMessage()
+            .isEqualTo("4 is not equal to the expected value 1. Accuracy: 2")
```

Comment:
Would this be even cleaner if you optional could pass the string into isMessage (likewise error for the other)? Or do you need them on separate lines so they fail differently.

Reply:
I want to avoid combining existing functionality into other operators. Equatability and type checking exist as operators, and the main driver of this design was to avoid doing just that.

Reply:
I do think there are compound operators (like the Assertions operators) that can take on a bit more responsibility and provide multiple assertions within a single operator. This felt like a case thought where it fit in nicely to the "do a single validation and return a new value" flow.

Reply:
In the case of IsMessage, is there anything else you'd do other than an isEqualTo? I agree that this felt a bit too modular.

Reply:
You could have both the modular version and the shortcut one with the string as a arg

Reply:
You could just not care about the contents of the message, which could be the case when you're writing operators for your project. Or you may want to do a non-equality check like looking for a substring or something.